### PR TITLE
[#10] Feat redux logger

### PR DESCRIPTION
**Pull request by e-schultz**

### CODE REVIEW

**Comment by e-schultz** [comment]
@igor-ka @SethDavenport could either of you review - just setting up the redux-logger. This is how I set it up for DBS the other week

File: src/store/configure-logger.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+declare let __DEV__: any;
+import * as Immutable from 'immutable';
```
Comment (igor-ka) [inline src/store/configure-logger.ts]: I dont have a strong opinion either way, but in theory we can import only a subset from Immutable that is needed, e.g. import {List} from 'immutable'

**Comment by igor-ka** [comment]
LGTM @e-schultz 
Let me know what you think about the * import from immutable, and otherwise :shipit:

**Comment by e-schultz** [comment]
@igor-ka changed to to just import {Iterable} - going to merge then

**Pull request merged**